runtime/obj.c

```diff
@@ -135,7 +138,7 @@ CAMLprim value caml_obj_block(value tag, value size)
 CAMLprim value caml_obj_with_tag(value new_tag_v, value arg)
 {
   CAMLparam2 (new_tag_v, arg);
-  CAMLlocal1 (res);
+  CAMLlocal2 (res, x);
```

Comment:
`x` is unused. Remove.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/cc8d6fe225c9fa655cc93ad93fb61561c6e9499e